### PR TITLE
fixed eslint errors and added tooling to catch those errors

### PR DESCRIPTION
**Pull request by Mozart409**
I love this starter and it's my go-to starter for nextjs, so I wanted to contribute.

## dependencies added
- prettier-standard
- propTypes

## Code I changed
- fixed security issues with <a href target="_blank" in Footer
- formatted and linted code (mostly double quotes to single) 
- added propType validation in Layout, _app.js
- added yarn lint and yarn format scripts to package.json
- added eslintrc to globally disable "react must be in scope" rule
- added prettierignore to ignore .next and node_modules
- bumped version to 1.0.6

"All matched files use Standard code style!"
grafik

### CODE REVIEW

**Comment by Mozart409** [comment]
Added alt tags to images for accessibility

**Comment by taylorbryant** [comment]
This is awesome! I’ve been debating with myself about adding Prettier/ESLint since I made this starter. I think it’s a good idea to add them!

Since you put this up, I wanted to ask: what do you think about adding husky and lint-staged to automatically lint and format newly committed code? You can see an example here: https://github.com/taylorbryant/project-boilerplate/blob/master/package.json#L39. It’d need to be a separate PR, but I was curious on your thoughts!

**Comment by Mozart409** [comment]
Thanks for your kind words. 😄 
I absolutely love husky and I use it all the time. I haven't heard about lint-staged, but I've been using [pretty-quick](https://github.com/azz/pretty-quick). From the first look both tools provide the same functionality. If you don't mind I will add husky and pretty-quick in a new PR.

**Comment by Mozart409** [comment]
I just found out, that you need to install prettier as a dependency to use pretty-quick. So I will remove pretty-quick and use prettier-standard for "pre-commit" formatting, because prettier-standard has an "--staged" option as well.

**Comment by Mozart409** [comment]
Hey,
is everything ok?

**Comment by taylorbryant** [comment]
@mozart409 Sorry, man. I’ve been a bit busy with work. I left a couple of comments. Can you take a look at those? Thanks!

**Comment by Mozart409** [comment]
@taylorbryant Hey no problem. 👍 . Ahm this may sound wired, but I can't find your comments? Where did you leave them?

**Comment by taylorbryant** [comment]
> @taylorbryant Hey no problem. 👍 . Ahm this may sound wired, but I can't find your comments? Where did you leave them?

I forgot to click "Submit review" 🤦‍♂️ You should be able to see them now

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true
+  },
+  extends: ['eslint:recommended', 'plugin:react/recommended'],
+  globals: {
+    Atomics: 'readonly',
```
[the remark this answers is not in the document]
Comment (Mozart409) [reply .eslintrc.js:9]: Yeah, I think you can safely remove the Atomics settings. I just copied it from another project of mine. I can't even remember why I have it in there.

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true
+    },
+    ecmaVersion: 2018,
+    sourceType: 'module'
+  },
+  plugins: ['react'],
+  rules: {
+    'react/react-in-jsx-scope': 0
```
[the remark this answers is not in the document]
Comment (Mozart409) [reply .eslintrc.js:21]: 👍 . yeah this rule is so annoying

File: package.json [hunk 3]
```diff
@@ -28,6 +30,8 @@
   "devDependencies": {
     "@fullhuman/postcss-purgecss": "2.1.0",
     "autoprefixer": "9.7.6",
+    "prettier-standard": "^16.2.1",
```
[the remark this answers is not in the document]
Comment (Mozart409) [reply package.json:33]: yeah. it combines eslint and prettier in one package with better defaults. I just added the .eslint config that my vscode picks up on the errors. Without it, I didn't get any visual warning.

**Pull request closed without merging**